definitions/npm/multimatch_v2.x.x/flow_v0.37.x-/multimatch_v2.x.x.js

```diff
@@ -0,0 +1,23 @@
+declare module "multimatch" {
+  declare type Options = {
+    debug: boolean,
+    dot: boolean,
+    flipNegate: boolean,
+    matchBase: boolean,
+    nobrace: boolean,
+    nocase: boolean,
+    nocommnet: boolean,
+    noext: boolean,
+    noglobstar: boolean,
+    nonegate: boolean,
+    nonull: boolean
+  };
+
+  declare function minimatch(
```

Comment:
`multimatch`?

Reply:
Ugh nice catch... I pivoted from `minimatch` to `multimatch` and left this behind.